src/lib/falcor/FalcorRouter.js

```diff
@@ -1,53 +1,55 @@
 import BaseRouter from "falcor-router"
 import { ghostArticleQuery } from 'lib/ghostAPI'
-import { sqlArticleQuery, sqlAuthorQuery } from 'lib/sql'
+import { dbAuthorQuery, dbArticleQuery, dbAuthorArticleQuery, dbInfoPagesQuery, dbArticleIssueQuery,
+dbArticleAuthorQuery, dbLatestIssueQuery, dbCategoryQuery, dbCategoryArticleQuery,
+dbFeaturedArticleQuery, dbEditorPickQuery, dbIssueCategoryQuery, dbIssueQuery } from 'lib/mariaDB'
 import falcor from 'falcor'
+import _ from 'lodash';
 
-const $ref = falcor.Model.ref; 
+const $ref = falcor.Model.ref;
 
-// FLAG TO CHECK IF YOU WANT TO ACCESS GHOST API AND DATABASES
-// THIS IS JUST FOR DEVELOPMENT WHILE WE DON'T HAVE ACTUAL SERVERS SET UP
-const USE_DATABASES = false; 
-
-// Transform ghost API names into our names
-const mapGhostNames = (name) => {
-  switch(name) {
-    case 'pubDate':
-      return 'published_at';
-    case 'body':
-      return 'markdown';
+let mapGhostNames = (falcorName) => {
+  switch (falcorName) {
+    case "teaser":
+      return "meta_description";
     default:
-      return name;
+      return falcorName;
   }
-};
+}
 
 export default class FalcorRouter extends BaseRouter.createClass([
   {
-    route: "appName",
-    get: () => {
-      return [{
-        path: ["appName"],
-        value: "The Gazelle",
-      }]
-    },
-  },
-  {
-    // Get author information from SQL database
-    // TODO: write path for articles written by the author
-    route: "authorsBySlug[{keys:slugs}]['name', 'photo', 'biography', 'slug']",
-    get: (pathSet) => {
-      return new Promise((resolve, reject) => {
-        if (!USE_DATABASES) {
-          resolve([]);
-          return null;
-        }
-        const query = {slug: {$in: pathSet.slugs}};
-        const length = pathSet.slugs.length;
-        const projection = {_id: 0};
-        pathSet[2].forEach((field) => {
-          projection[field] = 1;
+    route: "infoPages[{keys:slugs}]['title', 'html', 'slug']",
+    get: (pathSet) => {
+      return new Promise((resolve, reject) => {
+        dbInfoPagesQuery(pathSet.slugs, pathSet[2]).then((data) => {
```

Comment:
Generally with indexes like these, it's easier to pull out earlier in the code, to make it more readable like:

`requestedFields = pathSets[2]`